MdeModulePkg/Bus/Pci/XhciDxe/XhciSched.c

```diff
@@ -2175,7 +2176,7 @@ XhcInitializeDeviceSlot (
              XHC_GENERIC_TIMEOUT,
              (TRB_TEMPLATE **)(UINTN)&EvtTrb
              );
-  if (EFI_ERROR (Status)) {
+  if (EFI_ERROR (Status) || (EvtTrb == NULL)) {

```

Comment:
Which condition will cause the return status with EFI_SUCCESS and EvtTrb with NULL?

Reply:
Following the code through from XhcInitializeDeviceSlot, XhcCmdTransfer and to XhcExecTransfer I don't actually see how that can happen. I created this PR on behalf of a colleague who implemented this change.

Reply:
Could you let him provide some detail why this change is required?

Reply:
It looks like it might be possible for EFI_SUCCESS and a NULL EvtTrb to occur in XhcCheckUrbResult if Urb->Finished is TRUE, then EvtTrb isn't updated. Does that look correct to you?

Reply:
Yes. I have no other comment.